Maskbook/Scene/Maskbook/MessageResolver/WebPublicAPIMessageResolver+DB.swift

```diff
@@ -475,6 +475,9 @@ extension WebPublicApiMessageResolver {
             PersonaRepository.updatePersonaAvatar(identifier: identifier, avatar: avatar)
             return true
         }
+        // cache image for later usage
+        KingfisherManager.shared.retrieveImage(with: URL(string: avatar)!,
```

Comment:
As the persona data is handled by  `PersonaRepository`, the word to cache the avatar should be handled by  `PersonaRepository `.

Reply:
I don't think so,
PersonaRepository has a single function that manages the Persona entity. I don't want to add this to it.
`WebPublicAPIMessageResolver+DB` is fine and only need write once.

Reply:
I mean adding a new function to PersonaRepository, and move the Kingfisher logic to there .
Also, there is no `sendResponseToWebView(response: para.post, id: request.id)` called in `func storeAvatar(messageData: Data) -> Bool`, is it safe for the JS side to fulfill a promise?

Reply:
I will add a response like other update method.